cluster/gce/gci/configure-helper.sh

```diff
@@ -1453,48 +1453,76 @@ function create-master-etcd-apiserver-auth {
    fi
 }
 
+
 function docker-installed {
     if systemctl cat docker.service &> /dev/null ; then
         return 0
     else
         return 1
     fi
 }
+function addockeropt {
+	DOCKER_OPTS_FILE=/etc/docker/daemon.json
+	if [ "$#" -lt 1 ]; then
+	echo "No arguments are passed while adding docker options. Expect one argument"
+	exit 1
+	elif [ "$#" -gt 1 ]; then
+	echo "Only one argument is accepted"
+	exit 1
+	fi
+	# appends the given input to the docker opts file i.e. /etc/docker/daemon.json file
+	if [ -f "$DOCKER_OPTS_FILE" ]; then
+	cat >> "${DOCKER_OPTS_FILE}" <<EOF
+  $1
+EOF
+	else # if file is not present it creates a new file.
+	cat > "${DOCKER_OPTS_FILE}" <<EOF
+  {
```

Comment:
should $1 be written into the file here?

Reply:
actually this is not required. Removed the code as we are creating a file.